libraries/AP_Scripting/applets/VTOL-quicktune.lua

```diff
@@ -356,12 +366,16 @@ function update()
    end
 
    local sw_pos = quick_switch:get_aux_switch_pos()
+   if sw_pos == 1 and (not arming:is_armed() or not vehicle:get_likely_flying()) then
+      gcs:send_text(MAV_SEVERITY_EMERGENCY, string.format("Tuning: Must be flying to tune"))
```

Comment:
this will cause a flood of messages if the switch is changed when disarmed. If we need this message then we need to rate limit it

Reply:
originally I was thinking that flooding would be the attention getter, but will rate limit it...